src/playsim/p_effect.cpp

```diff
@@ -1277,9 +1295,3 @@ DEFINE_FIELD(DVisualThinker, PrevRoll);
 DEFINE_FIELD(DVisualThinker, Translation);
 DEFINE_FIELD(DVisualThinker, LightLevel);
 DEFINE_FIELD(DVisualThinker, cursector);
-DEFINE_FIELD(DVisualThinker, bXFlip);
-DEFINE_FIELD(DVisualThinker, bYFlip);
-DEFINE_FIELD(DVisualThinker, bDontInterpolate);
-DEFINE_FIELD(DVisualThinker, bAddLightLevel);
-DEFINE_FIELD(DVisualThinker, bFlipOffsetX);
-DEFINE_FIELD(DVisualThinker, bFlipOffsetY);
```

Comment:
Redirects might be needed for these much like accessing  an actor's `velx` to `vel.x`. Otherwise this could break backwards compatibility.

Reply:
they're already there in the FlagDefs